src/Orchard.Web/Modules/Orchard.OutputCache/Filters/OutputCacheFilter.cs

```diff
@@ -33,14 +32,12 @@ public class OutputCacheFilter : FilterProvider, IActionFilter, IResultFilter, I
         // Dependencies.
         private readonly ICacheManager _cacheManager;
         private readonly IOutputCacheStorageProvider _cacheStorageProvider;
-        private readonly ITagCache _tagCache;
-        private readonly IDisplayedContentItemHandler _displayedContentItemHandler;
         private readonly IWorkContextAccessor _workContextAccessor;
-        private readonly IThemeManager _themeManager;
```

Comment:
`ITagCache` and `IThemeManager` were already unused before this change, I just removed them to help tidy up.